extras/nginx-mtls-auth/README.md

```diff
@@ -0,0 +1,39 @@
+# Sample of using securing Model Server with nginx
```

Comment:
Add a reference to this doc from Security considerations sections from docker_container.md

Reply:
Added.